beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -142,21 +142,29 @@ func (s *Service) processFetchedDataRegSync(
 
 	blockReceiver := s.cfg.Chain.ReceiveBlock
 	invalidBlocks := 0
-	for _, blk := range data.blocks {
+	parentDoesNotExistBlocks := make([]int, 0, len(data.blocks))
```

Comment:
why not just make this a counter ? you aren't use the actual missing index values anywhere.